packages/synapse-react-client/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "rootDir": "src/lib",
     "outDir": "dist",
-    "module": "CommonJS",
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "Node",
```

Comment:
Emit ESM instead of CommonJS so it plays better with Vite. Only the apps use this output, so it should not be breaking.